scss/main.scss

```diff
@@ -1117,4 +1117,79 @@ p.cardText {
   width:100%;
   max-width: 395px;
   max-height: 261px;
-}
\ No newline at end of file
+}
+
+// ABOUT Section - What is CFA and Why join
+.aboutPage {
+  margin: 80px 96px;
+  max-width: 1440px;
+  @media(max-width: 768px) {
+    margin: 64px 48px;
+  }
+  @media(max-width: 360px) {
+    margin: 48px 24px;
+  }
+
+  .section {
+    padding-bottom: 48px;
+    @media (max-width: 651px) {
+      padding-bottom: 32px;
+    }
+      .infoContainer {
+        width: 100%;
+        max-width: 600px;
+        margin: 0 auto;
+        @media (max-width: 768px) {
+          max-width: 672px;
+        }
+        @media (max-width: 651px) {
+          max-width: 555px;
+        }
+        @media (max-width: 360px) {
+          max-width: 312px;
+        }
+        
+        h1 {
+          margin-bottom: 16px;
+          color: black;
+        }
+        .text {
+          font-size: 26px;
+          line-height: 34px;
+          padding-bottom: 24px;
+          @media (max-width: 651px) {
+            font-size: 18px;
+            line-height: 23px;
+          }
+        }
+        .learnMore {
+          margin-bottom: 48px;
+          @media (max-width: 651px) {
+            margin-bottom: 32px;
+          }
+          a {
+            font-family: "Work Sans",sans-serif;
```

Comment:
Let's use the `$font-family-alt` variable instead

Reply:
updated